src/Encamina.Enmarcha.AspNet.OpenApi/README.md

```diff
@@ -0,0 +1,66 @@
+﻿# OpenApi
+
+[![Nuget package](https://img.shields.io/nuget/v/Encamina.Enmarcha.AspNet.OpenApi)](https://www.nuget.org/packages/Encamina.Enmarcha.AspNet.OpenApi)
+
+This package provides extensions and types to work with OpenAPI (a.k.a. Swagger).
+
+## Setup
+
+### Nuget package
+
+First, [install NuGet](http://docs.nuget.org/docs/start-here/installing-nuget). Then, install [Encamina.Enmarcha.AspNet.OpenApi](https://www.nuget.org/packages/Encamina.Enmarcha.AspNet.OpenApi) from the package manager console:
+
+    PM> Install-Package Encamina.Enmarcha.AspNet.OpenApi
+
+### .NET CLI:
+
+First, [install .NET CLI](https://learn.microsoft.com/en-us/dotnet/core/tools/). Then, install [Encamina.Enmarcha.AspNet.OpenApi](https://www.nuget.org/packages/Encamina.Enmarcha.AspNet.OpenApi) from the .NET CLI:
+
+    dotnet add package Encamina.Enmarcha.AspNet.OpenApi
+
+## How to use
+
+### Middlewares
+
+#### GroupNameKeyAuthorizationMiddleware
+
+This middleware is used to block access to an OpenAPI document or specification b y using a pre-shared key that must be provided in the request header per specification.
```

Comment:
Perhaps we intended to write 'by' without a space.

Reply:
Fixed.